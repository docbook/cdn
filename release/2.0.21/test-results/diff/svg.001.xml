<html xmlns:deltaxml="http://www.deltaxml.com/ns/well-formed-delta-v1" xmlns="http://www.w3.org/1999/xhtml" xmlns:p0="http://www.w3.org/2000/svg" xmlns:dxx="http://www.deltaxml.com/ns/xml-namespaced-attribute" xmlns:dxa="http://www.deltaxml.com/ns/non-namespaced-attribute" deltaxml:deltaV2="A!=B" deltaxml:version="2.0" deltaxml:content-type="full-context">
   <head deltaxml:deltaV2="A=B">
      <link href="../../resources/css/db-prism.css" rel="stylesheet" type="text/css"/>
      <link rel="stylesheet" type="text/css" href="../../resources/css/default.css"/>
      <link href="../../resources/css/prism.css" rel="stylesheet" type="text/css"/>
      <meta charset="utf-8"/>
      <meta name="generator" content="DocBook XSL 2.0 Stylesheets Vxxx"/>
      <script type="text/javascript" src="../../resources/js/dbmodnizr.js"/>
      <title>Article wrapper</title>
   </head>
   <body deltaxml:deltaV2="A!=B">
      <article deltaxml:deltaV2="A!=B" id="R.1" class="article">
         <header deltaxml:deltaV2="A=B" class="article-titlepage">
            <h2>Article wrapper</h2>
         </header>
         <p deltaxml:deltaV2="A">This article contains two SVG diagrams.</p><deltaxml:textGroup deltaxml:deltaV2="A"><deltaxml:text deltaxml:deltaV2="A">
         </deltaxml:text></deltaxml:textGroup><div deltaxml:deltaV2="A!=B"><deltaxml:attributes deltaxml:deltaV2="A!=B"><dxa:class deltaxml:deltaV2="A!=B"><deltaxml:attributeValue deltaxml:deltaV2="A">mediaobject centerimg</deltaxml:attributeValue><deltaxml:attributeValue deltaxml:deltaV2="B">content</deltaxml:attributeValue></dxa:class><dxa:id deltaxml:deltaV2="A"><deltaxml:attributeValue deltaxml:deltaV2="A">R.1.3</deltaxml:attributeValue></dxa:id></deltaxml:attributes>
            <table deltaxml:deltaV2="A" border="0" summary="Manufactured viewport for HTML image" cellspacing="0" cellpadding="0" width="600">
               <tr style="height: 250px">
                  <td align="center">
                     <object data="test/graphics/svg-001.svg" type="image/svg+xml" height="250" alt="Example SVG polygons"/>
                  </td>
               </tr>
            </table><p deltaxml:deltaV2="B">This article contains two SVG diagrams.</p>
            <div deltaxml:deltaV2="A!=B"><deltaxml:attributes deltaxml:deltaV2="A!=B"><dxa:class deltaxml:deltaV2="A!=B"><deltaxml:attributeValue deltaxml:deltaV2="A">caption</deltaxml:attributeValue><deltaxml:attributeValue deltaxml:deltaV2="B">mediaobject centerimg</deltaxml:attributeValue></dxa:class><dxa:id deltaxml:deltaV2="B"><deltaxml:attributeValue deltaxml:deltaV2="B">R.1.3</deltaxml:attributeValue></dxa:id></deltaxml:attributes>
               <p deltaxml:deltaV2="A">Polygons (with fallback)</p><table deltaxml:deltaV2="B" border="0" summary="Manufactured viewport for HTML image" cellspacing="0" cellpadding="0" width="600">
                  <tr style="height: 250px">
                     <td align="center">
                        <object data="test/graphics/svg-001.svg" type="image/svg+xml" height="250" alt="Example SVG polygons"/>
                     </td>
                  </tr>
               </table><deltaxml:textGroup deltaxml:deltaV2="B"><deltaxml:text deltaxml:deltaV2="B">
               </deltaxml:text></deltaxml:textGroup><div deltaxml:deltaV2="B" class="caption">
                  <p>Polygons (with fallback)</p>
               </div>
            </div><deltaxml:textGroup deltaxml:deltaV2="B"><deltaxml:text deltaxml:deltaV2="B">
            </deltaxml:text></deltaxml:textGroup><div deltaxml:deltaV2="B" id="R.1.4" class="mediaobject centerimg">
               <table border="0" summary="Manufactured viewport for HTML image" cellspacing="0" cellpadding="0" width="600">
                  <tr style="height: 250px">
                     <td align="center">
                        <object data="test/graphics/svg-001.svg" type="image/svg+xml" height="250" alt="Example SVG polygons"/>
                     </td>
                  </tr>
               </table>
               <div class="caption">
                  <p>Polygons (without fallback)</p>
               </div>
            </div><deltaxml:textGroup deltaxml:deltaV2="B"><deltaxml:text deltaxml:deltaV2="B">
            </deltaxml:text></deltaxml:textGroup><div deltaxml:deltaV2="B" id="R.1.5" class="mediaobject">
               <table border="0" summary="Manufactured viewport for HTML image" cellspacing="0" cellpadding="0" width="600">
                  <tr style="height: 250px">
                     <td>
                        <p0:svg>
                           <p0:g transform="scale(0.5)">
                              <p0:polygon style="fill:red; stroke:blue; stroke-width:10" points="350, 75 379,161 469,161 397,215                          423,301 350,250 277,301 303,215                          231,161 321,161"/>
                              <p0:polygon style="fill:lime; stroke:blue; stroke-width:10" points="850,75  958,137.5 958,262.5                          850,325 742,262.6 742,137.5"/>
                           </p0:g>
                        </p0:svg>
                     </td>
                  </tr>
               </table>
               <div class="caption">
                  <p>Polygons (inline)</p>
               </div>
            </div>
         </div><deltaxml:textGroup deltaxml:deltaV2="A"><deltaxml:text deltaxml:deltaV2="A">
         </deltaxml:text></deltaxml:textGroup><div deltaxml:deltaV2="A" id="R.1.4" class="mediaobject centerimg">
            <table border="0" summary="Manufactured viewport for HTML image" cellspacing="0" cellpadding="0" width="600">
               <tr style="height: 250px">
                  <td align="center">
                     <object data="test/graphics/svg-001.svg" type="image/svg+xml" height="250" alt="Example SVG polygons"/>
                  </td>
               </tr>
            </table>
            <div class="caption">
               <p>Polygons (without fallback)</p>
            </div>
         </div><deltaxml:textGroup deltaxml:deltaV2="A"><deltaxml:text deltaxml:deltaV2="A">
         </deltaxml:text></deltaxml:textGroup><div deltaxml:deltaV2="A" id="R.1.5" class="mediaobject">
            <table border="0" summary="Manufactured viewport for HTML image" cellspacing="0" cellpadding="0" width="600">
               <tr style="height: 250px">
                  <td>
                     <p0:svg>
                        <p0:g transform="scale(0.5)">
                           <p0:polygon style="fill:red; stroke:blue; stroke-width:10" points="350, 75 379,161 469,161 397,215                          423,301 350,250 277,301 303,215                          231,161 321,161"/>
                           <p0:polygon style="fill:lime; stroke:blue; stroke-width:10" points="850,75  958,137.5 958,262.5                          850,325 742,262.6 742,137.5"/>
                        </p0:g>
                     </p0:svg>
                  </td>
               </tr>
            </table>
            <div class="caption">
               <p>Polygons (inline)</p>
            </div>
         </div>
      </article>
      <script deltaxml:deltaV2="A=B" src="../../resources/js/prism.js"/>
   </body>
</html>