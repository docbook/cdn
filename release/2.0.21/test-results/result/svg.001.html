<html xmlns="http://www.w3.org/1999/xhtml" xmlns:p="http://www.w3.org/ns/xproc" p:style=""><head><meta http-equiv="Content-Type" content="text/html; charset=UTF-8" /><title>Results for svg.001.xml</title><link rel="stylesheet" type="text/css" href="../../resources/css/default.css" /><script type="text/javascript" src="../../resources/js/dbmodnizr.js"></script><link rel="stylesheet" type="text/css" href="http://ajax.googleapis.com/ajax/libs/jqueryui/1.11.4/themes/start/jquery-ui.css" /><link rel="stylesheet" type="text/css" href="../../resources/css/prism.css" /><link rel="stylesheet" type="text/css" href="../../resources/css/db-prism.css" /><link href="../style/show-results.css" rel="stylesheet" type="text/css" /><meta name="deltaxml" content="true" /></head><body><h1>DocBook XSLT 2.0 Stylesheet output: svg.001.xml</h1><table class="diffs"><tbody><tr><th>Expected (<span class="exdiff">11</span> differences)</th><th>Actual (<span class="adiff">11</span> differences)</th></tr><tr><td><pre class="show-diff"><span class="tag diff">&lt;html&gt;
   <span class="tag nodiff">&lt;head&gt;
      <span class="tag">&lt;link <span class="att">href="../../resources/css/db-prism.css"</span> <span class="att">rel="stylesheet"</span> <span class="att">type="text/css"</span>/&gt;</span>
      <span class="tag">&lt;link <span class="att">href="../../resources/css/default.css"</span> <span class="att">rel="stylesheet"</span> <span class="att">type="text/css"</span>/&gt;</span>
      <span class="tag">&lt;link <span class="att">href="../../resources/css/prism.css"</span> <span class="att">rel="stylesheet"</span> <span class="att">type="text/css"</span>/&gt;</span>
      <span class="tag">&lt;meta <span class="att">charset="utf-8"</span>/&gt;</span>
      <span class="tag">&lt;meta <span class="att">content="DocBook XSL 2.0 Stylesheets Vxxx"</span> <span class="att">name="generator"</span>/&gt;</span>
      <span class="tag">&lt;script <span class="att">src="../../resources/js/dbmodnizr.js"</span> <span class="att">type="text/javascript"</span>/&gt;</span>
      <span class="tag">&lt;title&gt;Article wrapper&lt;/title&gt;</span>
   &lt;/head&gt;</span>
   <span class="tag diff">&lt;body&gt;
      <span class="tag diff">&lt;article <span class="att">class="article"</span> <span class="att">id="R.1"</span>&gt;
         <span class="tag nodiff">&lt;header <span class="att">class="article-titlepage"</span>&gt;
            <span class="tag">&lt;h2&gt;Article wrapper&lt;/h2&gt;</span>
         &lt;/header&gt;</span>
         <span class="tag diff del">&lt;p&gt;This article contains two SVG diagrams.&lt;/p&gt;</span><span class="diff txtchg">
         </span><span class="tag diff">&lt;div <span class="att">class="<span class="att attchg">mediaobject centerimg</span>"</span> <span class="att add">id="<span class="att">R.1.3</span>"</span>&gt;
            <span class="tag diff del">&lt;table <span class="att">border="0"</span> <span class="att">cellpadding="0"</span> <span class="att">cellspacing="0"</span> <span class="att">summary="Manufactured viewport for HTML image"</span> <span class="att">width="600"</span>&gt;
               <span class="tag">&lt;tr <span class="att">style="height: 250px"</span>&gt;
                  <span class="tag">&lt;td <span class="att">align="center"</span>&gt;
                     <span class="tag">&lt;object <span class="att">alt="Example SVG polygons"</span> <span class="att">data="test/graphics/svg-001.svg"</span> <span class="att">height="250"</span> <span class="att">type="image/svg+xml"</span>/&gt;</span>
                  &lt;/td&gt;</span>
               &lt;/tr&gt;</span>
            &lt;/table&gt;</span>
            <span class="tag diff">&lt;div <span class="att">class="<span class="att attchg">caption</span>"</span> <span class="att del">id="<span class="att"></span>"</span>&gt;
               <span class="tag diff del">&lt;p&gt;Polygons (with fallback)&lt;/p&gt;</span>
            &lt;/div&gt;</span>
         &lt;/div&gt;</span><span class="diff txtchg">
         </span><span class="tag diff del">&lt;div <span class="att">class="mediaobject centerimg"</span> <span class="att">id="R.1.4"</span>&gt;
            <span class="tag">&lt;table <span class="att">border="0"</span> <span class="att">cellpadding="0"</span> <span class="att">cellspacing="0"</span> <span class="att">summary="Manufactured viewport for HTML image"</span> <span class="att">width="600"</span>&gt;
               <span class="tag">&lt;tr <span class="att">style="height: 250px"</span>&gt;
                  <span class="tag">&lt;td <span class="att">align="center"</span>&gt;
                     <span class="tag">&lt;object <span class="att">alt="Example SVG polygons"</span> <span class="att">data="test/graphics/svg-001.svg"</span> <span class="att">height="250"</span> <span class="att">type="image/svg+xml"</span>/&gt;</span>
                  &lt;/td&gt;</span>
               &lt;/tr&gt;</span>
            &lt;/table&gt;</span>
            <span class="tag">&lt;div <span class="att">class="caption"</span>&gt;
               <span class="tag">&lt;p&gt;Polygons (without fallback)&lt;/p&gt;</span>
            &lt;/div&gt;</span>
         &lt;/div&gt;</span><span class="diff txtchg">
         </span><span class="tag diff del">&lt;div <span class="att">class="mediaobject"</span> <span class="att">id="R.1.5"</span>&gt;
            <span class="tag">&lt;table <span class="att">border="0"</span> <span class="att">cellpadding="0"</span> <span class="att">cellspacing="0"</span> <span class="att">summary="Manufactured viewport for HTML image"</span> <span class="att">width="600"</span>&gt;
               <span class="tag">&lt;tr <span class="att">style="height: 250px"</span>&gt;
                  <span class="tag">&lt;td&gt;
                     <span class="tag">&lt;p0:svg&gt;
                        <span class="tag">&lt;p0:g <span class="att">transform="scale(0.5)"</span>&gt;
                           <span class="tag">&lt;p0:polygon <span class="att">points="350, 75 379,161 469,161 397,215                          423,301 350,250 277,301 303,215                          231,161 321,161"</span> <span class="att">style="fill:red; stroke:blue; stroke-width:10"</span>/&gt;</span>
                           <span class="tag">&lt;p0:polygon <span class="att">points="850,75  958,137.5 958,262.5                          850,325 742,262.6 742,137.5"</span> <span class="att">style="fill:lime; stroke:blue; stroke-width:10"</span>/&gt;</span>
                        &lt;/p0:g&gt;</span>
                     &lt;/p0:svg&gt;</span>
                  &lt;/td&gt;</span>
               &lt;/tr&gt;</span>
            &lt;/table&gt;</span>
            <span class="tag">&lt;div <span class="att">class="caption"</span>&gt;
               <span class="tag">&lt;p&gt;Polygons (inline)&lt;/p&gt;</span>
            &lt;/div&gt;</span>
         &lt;/div&gt;</span>
      &lt;/article&gt;</span>
      <span class="tag nodiff">&lt;script <span class="att">src="../../resources/js/prism.js"</span>/&gt;</span>
   &lt;/body&gt;</span>
&lt;/html&gt;</span></pre></td><td><pre class="show-diff"><span class="tag diff">&lt;html&gt;
   <span class="tag nodiff">&lt;head&gt;
      <span class="tag">&lt;link <span class="att">href="../../resources/css/db-prism.css"</span> <span class="att">rel="stylesheet"</span> <span class="att">type="text/css"</span>/&gt;</span>
      <span class="tag">&lt;link <span class="att">href="../../resources/css/default.css"</span> <span class="att">rel="stylesheet"</span> <span class="att">type="text/css"</span>/&gt;</span>
      <span class="tag">&lt;link <span class="att">href="../../resources/css/prism.css"</span> <span class="att">rel="stylesheet"</span> <span class="att">type="text/css"</span>/&gt;</span>
      <span class="tag">&lt;meta <span class="att">charset="utf-8"</span>/&gt;</span>
      <span class="tag">&lt;meta <span class="att">content="DocBook XSL 2.0 Stylesheets Vxxx"</span> <span class="att">name="generator"</span>/&gt;</span>
      <span class="tag">&lt;script <span class="att">src="../../resources/js/dbmodnizr.js"</span> <span class="att">type="text/javascript"</span>/&gt;</span>
      <span class="tag">&lt;title&gt;Article wrapper&lt;/title&gt;</span>
   &lt;/head&gt;</span>
   <span class="tag diff">&lt;body&gt;
      <span class="tag diff">&lt;article <span class="att">class="article"</span> <span class="att">id="R.1"</span>&gt;
         <span class="tag nodiff">&lt;header <span class="att">class="article-titlepage"</span>&gt;
            <span class="tag">&lt;h2&gt;Article wrapper&lt;/h2&gt;</span>
         &lt;/header&gt;</span>
         <span class="tag diff">&lt;div <span class="att">class="<span class="att attchg">content</span>"</span> <span class="att del">id="<span class="att"></span>"</span>&gt;
            <span class="tag diff add">&lt;p&gt;This article contains two SVG diagrams.&lt;/p&gt;</span>
            <span class="tag diff">&lt;div <span class="att">class="<span class="att attchg">mediaobject centerimg</span>"</span> <span class="att add">id="<span class="att">R.1.3</span>"</span>&gt;
               <span class="tag diff add">&lt;table <span class="att">border="0"</span> <span class="att">cellpadding="0"</span> <span class="att">cellspacing="0"</span> <span class="att">summary="Manufactured viewport for HTML image"</span> <span class="att">width="600"</span>&gt;
                  <span class="tag">&lt;tr <span class="att">style="height: 250px"</span>&gt;
                     <span class="tag">&lt;td <span class="att">align="center"</span>&gt;
                        <span class="tag">&lt;object <span class="att">alt="Example SVG polygons"</span> <span class="att">data="test/graphics/svg-001.svg"</span> <span class="att">height="250"</span> <span class="att">type="image/svg+xml"</span>/&gt;</span>
                     &lt;/td&gt;</span>
                  &lt;/tr&gt;</span>
               &lt;/table&gt;</span><span class="diff txtchg">
               </span><span class="tag diff add">&lt;div <span class="att">class="caption"</span>&gt;
                  <span class="tag">&lt;p&gt;Polygons (with fallback)&lt;/p&gt;</span>
               &lt;/div&gt;</span>
            &lt;/div&gt;</span><span class="diff txtchg">
            </span><span class="tag diff add">&lt;div <span class="att">class="mediaobject centerimg"</span> <span class="att">id="R.1.4"</span>&gt;
               <span class="tag">&lt;table <span class="att">border="0"</span> <span class="att">cellpadding="0"</span> <span class="att">cellspacing="0"</span> <span class="att">summary="Manufactured viewport for HTML image"</span> <span class="att">width="600"</span>&gt;
                  <span class="tag">&lt;tr <span class="att">style="height: 250px"</span>&gt;
                     <span class="tag">&lt;td <span class="att">align="center"</span>&gt;
                        <span class="tag">&lt;object <span class="att">alt="Example SVG polygons"</span> <span class="att">data="test/graphics/svg-001.svg"</span> <span class="att">height="250"</span> <span class="att">type="image/svg+xml"</span>/&gt;</span>
                     &lt;/td&gt;</span>
                  &lt;/tr&gt;</span>
               &lt;/table&gt;</span>
               <span class="tag">&lt;div <span class="att">class="caption"</span>&gt;
                  <span class="tag">&lt;p&gt;Polygons (without fallback)&lt;/p&gt;</span>
               &lt;/div&gt;</span>
            &lt;/div&gt;</span><span class="diff txtchg">
            </span><span class="tag diff add">&lt;div <span class="att">class="mediaobject"</span> <span class="att">id="R.1.5"</span>&gt;
               <span class="tag">&lt;table <span class="att">border="0"</span> <span class="att">cellpadding="0"</span> <span class="att">cellspacing="0"</span> <span class="att">summary="Manufactured viewport for HTML image"</span> <span class="att">width="600"</span>&gt;
                  <span class="tag">&lt;tr <span class="att">style="height: 250px"</span>&gt;
                     <span class="tag">&lt;td&gt;
                        <span class="tag">&lt;p0:svg&gt;
                           <span class="tag">&lt;p0:g <span class="att">transform="scale(0.5)"</span>&gt;
                              <span class="tag">&lt;p0:polygon <span class="att">points="350, 75 379,161 469,161 397,215                          423,301 350,250 277,301 303,215                          231,161 321,161"</span> <span class="att">style="fill:red; stroke:blue; stroke-width:10"</span>/&gt;</span>
                              <span class="tag">&lt;p0:polygon <span class="att">points="850,75  958,137.5 958,262.5                          850,325 742,262.6 742,137.5"</span> <span class="att">style="fill:lime; stroke:blue; stroke-width:10"</span>/&gt;</span>
                           &lt;/p0:g&gt;</span>
                        &lt;/p0:svg&gt;</span>
                     &lt;/td&gt;</span>
                  &lt;/tr&gt;</span>
               &lt;/table&gt;</span>
               <span class="tag">&lt;div <span class="att">class="caption"</span>&gt;
                  <span class="tag">&lt;p&gt;Polygons (inline)&lt;/p&gt;</span>
               &lt;/div&gt;</span>
            &lt;/div&gt;</span>
         &lt;/div&gt;</span>
      &lt;/article&gt;</span>
      <span class="tag nodiff">&lt;script <span class="att">src="../../resources/js/prism.js"</span>/&gt;</span>
   &lt;/body&gt;</span>
&lt;/html&gt;</span></pre></td></tr><tr><td><em>The “current” CSS styling is applied
                  to the this entire document; the results in this
                  column may be anomalous. Also, image links are broken
                  in this column.</em></td><td> </td></tr><tr><td><article id="R.1" class="article"><header class="article-titlepage"><h2>Article wrapper</h2></header><p>This article contains two SVG diagrams.</p><div id="R.1.3" class="mediaobject centerimg"><table border="0" summary="Manufactured viewport for HTML image" cellspacing="0" cellpadding="0" width="600"><tr style="height: 250px"><td align="center"><object data="test/graphics/svg-001.svg" type="image/svg+xml" height="250" alt="Example SVG polygons"></object></td></tr></table><div class="caption"><p>Polygons (with fallback)</p></div></div><div id="R.1.4" class="mediaobject centerimg"><table border="0" summary="Manufactured viewport for HTML image" cellspacing="0" cellpadding="0" width="600"><tr style="height: 250px"><td align="center"><object data="test/graphics/svg-001.svg" type="image/svg+xml" height="250" alt="Example SVG polygons"></object></td></tr></table><div class="caption"><p>Polygons (without fallback)</p></div></div><div id="R.1.5" class="mediaobject"><table border="0" summary="Manufactured viewport for HTML image" cellspacing="0" cellpadding="0" width="600"><tr style="height: 250px"><td><svg xmlns="http://www.w3.org/2000/svg">
  <g transform="scale(0.5)">
    <polygon style="fill:red; stroke:blue; stroke-width:10" points="350, 75 379,161 469,161 397,215                          423,301 350,250 277,301 303,215                          231,161 321,161"></polygon>
    <polygon style="fill:lime; stroke:blue; stroke-width:10" points="850,75  958,137.5 958,262.5                          850,325 742,262.6 742,137.5"></polygon>
  </g>
</svg></td></tr></table><div class="caption"><p>Polygons (inline)</p></div></div></article><script src="../../resources/js/prism.js"></script></td><td><article id="R.1" class="article"><header class="article-titlepage"><h2>Article wrapper</h2></header><div class="content"><p>This article contains two SVG diagrams.</p><div id="R.1.3" class="mediaobject centerimg"><table border="0" summary="Manufactured viewport for HTML image" cellspacing="0" cellpadding="0" width="600"><tr style="height: 250px"><td align="center"><object data="test/graphics/svg-001.svg" type="image/svg+xml" height="250" alt="Example SVG polygons"></object></td></tr></table><div class="caption"><p>Polygons (with fallback)</p></div></div><div id="R.1.4" class="mediaobject centerimg"><table border="0" summary="Manufactured viewport for HTML image" cellspacing="0" cellpadding="0" width="600"><tr style="height: 250px"><td align="center"><object data="test/graphics/svg-001.svg" type="image/svg+xml" height="250" alt="Example SVG polygons"></object></td></tr></table><div class="caption"><p>Polygons (without fallback)</p></div></div><div id="R.1.5" class="mediaobject"><table border="0" summary="Manufactured viewport for HTML image" cellspacing="0" cellpadding="0" width="600"><tr style="height: 250px"><td><svg xmlns="http://www.w3.org/2000/svg">
  <g transform="scale(0.5)">
    <polygon style="fill:red; stroke:blue; stroke-width:10" points="350, 75 379,161 469,161 397,215                          423,301 350,250 277,301 303,215                          231,161 321,161"></polygon>
    <polygon style="fill:lime; stroke:blue; stroke-width:10" points="850,75  958,137.5 958,262.5                          850,325 742,262.6 742,137.5"></polygon>
  </g>
</svg></td></tr></table><div class="caption"><p>Polygons (inline)</p></div></div></div></article><script src="../../resources/js/prism.js"></script></td></tr></tbody></table><h2>XML source</h2><pre data-src="../src/svg.001.xml"></pre></body><script type="text/javascript" src="http://code.jquery.com/jquery-1.6.4.min.js"></script><script type="text/javascript" src="http://ajax.googleapis.com/ajax/libs/jqueryui/1.11.4/jquery-ui.min.js"></script><script type="text/javascript" src="../../resources/js/annotation.js"></script><script type="text/javascript" src="../../resources/js/prism.js"></script></html>