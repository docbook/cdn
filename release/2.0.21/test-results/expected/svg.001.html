<html xmlns="http://www.w3.org/1999/xhtml"><head><meta http-equiv="Content-Type" content="text/html; charset=UTF-8" /><title>Article wrapper</title><meta charset="utf-8" /><meta name="generator" content="DocBook XSL 2.0 Stylesheets V2.0.17" /><link rel="stylesheet" type="text/css" href="../../resources/css/default.css" /><script type="text/javascript" src="../../resources/js/dbmodnizr.js"></script><link href="../../resources/css/prism.css" rel="stylesheet" type="text/css" /><link href="../../resources/css/db-prism.css" rel="stylesheet" type="text/css" /></head><body><article id="R.1" class="article"><header class="article-titlepage"><h2>Article wrapper</h2></header><p>This article contains two SVG diagrams.</p><div id="R.1.3" class="mediaobject centerimg"><table border="0" summary="Manufactured viewport for HTML image" cellspacing="0" cellpadding="0" width="600"><tr style="height: 250px"><td align="center"><object data="test/graphics/svg-001.svg" type="image/svg+xml" height="250" alt="Example SVG polygons"></object></td></tr></table><div class="caption"><p>Polygons (with fallback)</p></div></div><div id="R.1.4" class="mediaobject centerimg"><table border="0" summary="Manufactured viewport for HTML image" cellspacing="0" cellpadding="0" width="600"><tr style="height: 250px"><td align="center"><object data="test/graphics/svg-001.svg" type="image/svg+xml" height="250" alt="Example SVG polygons"></object></td></tr></table><div class="caption"><p>Polygons (without fallback)</p></div></div><div id="R.1.5" class="mediaobject"><table border="0" summary="Manufactured viewport for HTML image" cellspacing="0" cellpadding="0" width="600"><tr style="height: 250px"><td><svg xmlns="http://www.w3.org/2000/svg">
  <g transform="scale(0.5)">
    <polygon style="fill:red; stroke:blue; stroke-width:10" points="350, 75 379,161 469,161 397,215                          423,301 350,250 277,301 303,215                          231,161 321,161"></polygon>
    <polygon style="fill:lime; stroke:blue; stroke-width:10" points="850,75  958,137.5 958,262.5                          850,325 742,262.6 742,137.5"></polygon>
  </g>
</svg></td></tr></table><div class="caption"><p>Polygons (inline)</p></div></div></article><script src="../../resources/js/prism.js"></script></body></html>